pkg/types/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package types defines the InstallConfig that contains user-tuneable options.
```

Comment:
nit: Are other types going to go in here too?  If it's just `InstallConfig`, maybe we should move that type definition into `pkg/asset/installconfig`.

Reply:
I'm inclined to keep it here for now, as the `pkg/asset` is more for generating assets, but the InstallConfig is kinda exportable, and might be referenced by other projects like cluster-operator, etc. So I want to make it stay in a top level package for easier looking up.